ROBERTS_NOTES.txt

```diff
@@ -0,0 +1,72 @@
+Setup
+Copied keys/token in
+added TWITTER env variables to .env file
+Modified docker-compose.yml to pull files locally
+cp ../SHARED/constants.json to WEB
+* Add 'python' endpoint for interactive
+
+GIT
+Create a new branch
+git checkout -b pep8_mods origin/master
+
+Look at the modifications
+git status
+
+Must "add" changes to files to the staging area:
+git add controllers/*.py
+
+Commit to the staging area
+git commit -m "just a couple style changes"
+
+Push to github
+git push origin pep8_mods
+
+After one or more changes, put in a pull request into main.
+
+APScheduler
+BackgroundScheduler
+MemoryJobStore
+ThreadPoolExecutor
+
+How to build and start the docker
+csh
+docker-compose build
+source .env  **
+docker-compose up -d --build
+
+# Enter shell
+docker exec -it sms-flask /bin/bash
+
+
+TODO:  SHould we put a USER line in DOCKERFILE?
+
+2019-07-15
+git clone https://github.com/robertlugg/symptomsurvey_backend.git
+git checkout -b pep8_mods origin/pep8_mods
+
+** Install this to automaticly source .env
+pip install python-dotenv
+
+
+git clone https://github.com/robertlugg/symptomsurvey_backend.git
+git checkout -b twitter_service origin/master
+
+https://docs.mongodb.com/manual/reference/mongo-shell/
+
+get into mongodb container
+docker exec -it sms-mongodb bash
+mongo
+show dbs
+use <new_dbs>
+show collections
+db.<collection>.find({})
+db.<collection>.count()
+
+Docker container other stuff
+To install other packages, update first:
+apt-get update
+then search:
+apt-cache search things
+apt-get install procps
+apt-get install iputils-ping
+apt-get install htop
```

Comment:
If we want to retain this info we should fold it into an .md file. It can either go into the README or can be linked from there.

Reply:
This is my mistake.  I did not mean to check this in.